device/protocol_mqtt_v1.go

```diff
@@ -326,6 +326,13 @@ func (d *Device) SetProperty(interfaceName, path string, value interface{}) erro
 		if err := interfaces.ValidateIndividualMessage(iface, path, value); err != nil {
 			return err
 		}
+
+		// Check if we're resending the same property twice
+		if d.hasAlreadySentPropertyValue(interfaceName, path, value) {
+			// in that case, do not enqueue the message
+			fmt.Printf("Property %s%s already set on the same value, not resending it\n", interfaceName, path)
```

Comment:
Leaving this as a comment - this is an SDK, as such we should avoid printing to stdout and instead we should document the behavior. Given the current development spree, let's do it outside of the scope of this PR